### DISCUSSION TITLE
Micronaut resilience4j 2.1.0 pointing to micronaut 3.7.3, not 4.x

### DISCUSSION THREAD

**Discussion by subash89** [resilience4j/resilience4j#2011] [Q&A]
Hi, 
We're on micronaut 4.x the latest. Micronaut 3.x to 4.x is a major version change and those two versions not compatible. So we cannot get resilience4j-micronaut 2.1.0 maven dependency working in micronaut 4.x app. Because it uses 3.7.3 micronaut parent bom. Is there anyone able to make it work with micronaut 4.x app ?

**Comment by RobWin** [comment]
Hi, you could create a PR and provide a micronaut 4.x compatible module.
Resilience4j is a community project and needs contributions to be able to stay up-to-date.
